packages/eos-components/src/components/Header.vue

```diff
@@ -30,16 +30,22 @@
 
   .header {
     background: white;
+    height: $navbar-height;
   }
 
   .header > .container {
     padding: $card-spacer-x;
   }
 
   $logo-size: 50px;
+  // 2.27 is the AR of the endless logo
+  $logo-height: $logo-size / 2.27;
```

Comment:
Nit: if we now have $logo-height then the $logo-size is actually a $logo-width. Good variable names are also documentation, so you could do:
```
  $logo-width: 50px;
  $logo-aspect-ratio: 2.27;
  $logo-height: $logo-width / logo-aspect-ratio;
```